packetbeat/beater/packetbeat.go

```diff
@@ -49,11 +49,6 @@ type CmdLineArgs struct {
 
 var cmdLineArgs CmdLineArgs
 
-const (
-	defaultQueueSize     = 2048
```

Comment:
no need to mention internal default value. This only reduces buffering a little.